ParabolInc/parabol
# feat(Team Insights): Add most used retro templates

# Description

Fixes #8467 

We accumulate the used templates from the past 1 year of use at the end of a meeting and show them in insights. I've chosen 1 year as there are a lot of teams running retros only once a month, so 30 days like for the other insights is too short of a period.

## Demo

https://www.loom.com/share/495b2ad2232f477d9d68bbff81c219ce?sid=464ce1c3-6f36-4764-8ad7-3570f91e0d97

## Testing scenarios

- enable `teamInsights` org flag
- finish any meeting
- when there was at least 1 retro run in the last year, see top retro templates appear in insights
- run a retro with another template and see it affecting the insight

## Final checklist
